ark/utils/metacluster_remap_gui/test_colormap_helper.py

```diff
@@ -0,0 +1,9 @@
+from .colormap_helper import distinct_cmap, distinct_rgbs
```

Comment:
for test files, please name them original_file_name_test.py

Reply:
Fixed, I like this, makes it sort the test file right next to implementation. Cool to see pytest added this to suffix to file collection in 2018.